src/app/effects/collection.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { EffectsTestingModule, EffectsRunner } from '@ngrx/effects/testing';
```

Comment:
@krjordan Sorry about missing out the RxJs imports. Very amateur. Not sure how it was working *on my machine. I fixed it.